docs/src/Groups/groups.md

```diff
@@ -285,7 +434,112 @@ f1*f2
+det(x::MatrixGroupElem)
+trace(x::MatrixGroupElem)
+multiplicative_jordan_decomposition(x::MatrixGroupElem)
+issemisimple(x::MatrixGroupElem{T}) where T <: FinFieldElem
+isunipotent(x::MatrixGroupElem{T}) where T <: FinFieldElem
+```
+
+### Sesquilinear forms
+
+```@docs
+SesquilinearForm{T<:RingElem}
+isalternating_form(f::SesquilinearForm)
+ishermitian_form(f::SesquilinearForm)
+isquadratic_form(f::SesquilinearForm)
+issymmetric_form(f::SesquilinearForm)
+alternating_form(B::MatElem{T}) where T <: FieldElem
+symmetric_form(B::MatElem{T}) where T <: FieldElem
+hermitian_form(B::MatElem{T}) where T <: FieldElem
+quadratic_form(B::MatElem{T}) where T <: FieldElem
+quadratic_form(f::MPolyElem{T}) where T <: FieldElem
+corresponding_bilinear_form(B::SesquilinearForm)
+corresponding_quadratic_form(B::SesquilinearForm)
+gram_matrix(f::SesquilinearForm)
+defining_polynomial(f::SesquilinearForm)
+radical(f::SesquilinearForm{T}) where T
+witt_index(f::SesquilinearForm{T}) where T
+isdegenerate(f::SesquilinearForm{T}) where T
+issingular(f::SesquilinearForm{T}) where T
+iscongruent(f::SesquilinearForm{T}, g::SesquilinearForm{T}) where T <: RingElem
+```
+
+### Invariant forms
+
+```@docs
+invariant_bilinear_forms(G::MatrixGroup{S,T}) where {S,T}
+invariant_sesquilinear_forms(G::MatrixGroup{S,T}) where {S,T}
+invariant_quadratic_forms(G::MatrixGroup{S,T}) where {S,T}
+invariant_symmetric_forms(G::MatrixGroup{S,T}) where {S,T}
+invariant_alternating_forms(G::MatrixGroup{S,T}) where {S,T}
+invariant_hermitian_forms(G::MatrixGroup{S,T}) where {S,T}
+invariant_bilinear_form(G::MatrixGroup)
+invariant_sesquilinear_form(G::MatrixGroup)
+invariant_quadratic_form(G::MatrixGroup)
+preserved_quadratic_forms(G::MatrixGroup{S,T}) where {S,T}
+preserved_sesquilinear_forms(G::MatrixGroup{S,T}) where {S,T}
+isometry_group(f::SesquilinearForm{T}) where T
+```
+
+### Utilities for matrices (replace by available functions, or document elsewhere?)
+
+```@docs
+pol_elementary_divisors(A::MatElem{T}) where T
+generalized_jordan_block(f::T, n::Int) where T<:PolyElem
+generalized_jordan_form(A::MatElem{T}; with_pol=false) where T
+matrix(A::Array{AbstractAlgebra.Generic.FreeModuleElem{T},1}) where T<: FieldElem
+upper_triangular_matrix(L)
+lower_triangular_matrix(L)
+conjugate_transpose(x::MatElem{T}) where T <: FinFieldElem
+complement(V::AbstractAlgebra.Generic.FreeModule{T}, W::AbstractAlgebra.Generic.Submodule{T}) where T <: FieldElem
+permutation_matrix(F::Ring, Q::AbstractVector{T}) where T <: Base.Integer
+isskewsymmetric_matrix(B::MatElem{T}) where T <: RingElem
+ishermitian_matrix(B::MatElem{T}) where T <: FinFieldElem
+
+
+submatrix(A::MatElem, i::Int, j::Int, nr::Int, nc::Int)
+insert_block(A::MatElem{T}, B::MatElem{T}, i::Int, j::Int) where T <: RingElem
+insert_block!(A::MatElem{T}, B::MatElem{T}, i::Int, j::Int) where T <: RingElem
+diagonal_join(V::AbstractVector{T}) where T <: MatElem
+block_matrix(m::Int, n::Int, V::AbstractVector{T}) where T <: MatElem
```

Comment:
There are in the process of being removed (see PR #425 which I can hopefully merge today). So let's not add them here
```suggestion
```